webpki-ccadb/src/lib.rs

```diff
@@ -0,0 +1,252 @@
+use std::cmp::Ordering;
+use std::collections::{BTreeMap, HashSet};
+
+use chrono::{NaiveDate, Utc};
+use num_bigint::BigUint;
+use pki_types::CertificateDer;
+use serde::Deserialize;
+
+// Fetch root certificate data from the CCADB server.
+//
+// Returns an ordered BTreeMap of the root certificates, keyed by the SHA256 fingerprint of the
+// certificate. Panics if there are any duplicate fingerprints.
+pub async fn fetch_ccadb_roots() -> BTreeMap<String, CertificateMetadata> {
```

Comment:
Before we expose this as public API surface WDYT about making it return a `Result`? 

Panicing for failures made sense in the original context where this was being driven from a unit test in CI, but perhaps other consumers would prefer to handle errors.

Reply:
My thoughts are that it's ok for this function to panic since it is intended to be used from a non-interactive workflow. A consumer that wanted to load the certificates at runtime should use the rustls-native-certs crate. If you prefer it to return a Result then I will make the change.

Reply:
>  A consumer that wanted to load the certificates at runtime should use the rustls-native-certs crate.

Presumably if someone is looking at this crate they're specifically interested in the Mozilla roots, and roots that will differ based on platform.

> If you prefer it to return a Result then I will make the change.

If the code as-is meets your needs then I don't feel strongly :+1:

Reply:
Thanks.